### ISSUE TITLE
Not sure what is not working in my "simple" program

### ISSUE THREAD

**Issue by C3R3S1** [PSS-Tools-Development/pssapi.py#72]
I am trying to get ship layouts by id but I keep getting the error "PssApiError" I'm not sure what is wrong in my code. Any help would be appreciated. I have attached my code and the console output.
![image](https://github.com/user-attachments/assets/c77aaa97-15e2-4ba7-92c6-9728ed2ab952)
![image](https://github.com/user-attachments/assets/a5c27b6e-9ca6-45fb-8f87-e550c26c0bda)

**Comment by Zukunftsmusik** [comment]
Hey there! You need an access token for `ship_service.get_ship_by_user_id`, like in the [showcase.py, line 45](https://github.com/PSS-Tools-Development/pssapi.py/blob/14706081ab35828d7dd59bfbc7822592f873a170/examples/showcase.py#L45), but you've passed a `device_key` instead. To get an access token, you'll need a `checksum_key` parameter, which we aren't allowed to share with this library unfortunately.

**Comment by C3R3S1** [comment]
Oh ok, how can I go about getting a checksum key? I understand you can’t share it but is there a way to get one from savy?

**Comment by Zukunftsmusik** [comment]
Copying from Issue #69 cause I'm lazy :D 
> Hey there, the `checksum_key` parameter is a "secret" that we can't share with the library at this point (we don't have the permission). However it's not that secret anymore and you can find it on github in other repos. E.g. search for **Pixel Starships DeviceLogin** ;)

**Comment by C3R3S1** [comment]
Yeah Ive been looking around but I feel like people have wiped it from there repos, but Ill keep looking, thanks!

**Comment by C3R3S1** [comment]
Figured it out thanks!

**Comment by bluejack** [comment]
@C3R3S1 -- if you don't mind messaging me, I have been looking into how this is generated or obtained and have not followed your footsteps well enough. feel free to email me bluujack at gmail, or leave me another clue here...

**Comment by bluejack** [comment]
nm, got myself sorted.
